### PR TITLE
Kevin/coinsuper

### PR DESCRIPTION
**Pull request by kevinobama**
hi,I am so glad that I have done coinsuper API for x change.  I am ready to merge.  I will continue to add unit test later.

### CODE REVIEW

File: xchange-coinsuper/pom.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <!-- Parent provides default configuration for dependencies -->
+    <dependencies>
+
+        <dependency>
+            <groupId>org.knowm.xchange</groupId>
+            <artifactId>xchange-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
```
Comment (timmolter) [inline xchange-coinsuper/pom.xml]: Stick to Jackson json please.

**Comment by timmolter** [comment]
Please make this dependent on Jackson JSON as to not add additional dependency.

File: xchange-coinsuper/pom.xml (hunk 1 above)
Comment (kevinobama) [reply xchange-coinsuper/pom.xml]: thanks for responding.I'll update it as soon as possible.

**Comment by kevinobama** [comment]
U have updated it.

**Pull request merged**